lib/Language/en/translation.json

```diff
@@ -1297,8 +1298,11 @@
       "bulkGeocoderErrorMessage": "Unable to map addresses to lat-long coordinates, as an error occurred while retrieving address coordinates. Please check your internet connection or try again later.",
       "legendZFilterLabel": "Extreme values",
       "legendNullLabel": "(No value)",
+      "styleDisabledLabel": "Disable style",
```

Comment:
Hmm, to me Display Variable: Disable style doesn't read very well. Furthermore, in the bike path example, selecting disable style does more than just disabling the style, it shows all the data. I would suggest that it be reworded to "Show all" with the disabling of the style being a consequence of being unable to cram multiple legends into showing all the data

Reply:
I agree, I don't like the wording very much.

But in most cases, if style is disabled, you won't seen any more features, everything will just be the same color.
So I'm not sure about "Show all" as an alternative.

What about "No style" or "None"?

Reply:
I'd prefer "Disable style" over "None" since Display Variable: None reads like nothing should be shown on the screen. Perhaps this should be a checkbox option instead of adding it to the drop down but we can always do that later